### PR TITLE
Add support for contacts array

### PR DESCRIPTION
**Pull request by daganomri**
While working with this repo I noticed that the contactsArrayMessage was already typed but not integrated in the message flow.
This PR adds contactsArrayMessage to the message flow.

### CODE REVIEW

File: Example/example.ts [hunk 1]
```diff
@@ -93,6 +93,10 @@ async function example() {
         } else if (messageType === MessageType.contact) {
             const contact = m.message.contactMessage
             console.log(sender + ' sent contact (' + contact.displayName + '): ' + contact.vcard)
+        } else if (messageType === MessageType.contactsArray) {
+            const contacts = m.message.contactsArrayMessage
+            console.log(sender + ` sent ${contacts.displayName}:`)
+            contacts.contacts.forEach(contact => console.log(`${contact.displayName}: ${contact.vcard}`))
```
Comment (adiwajshing) [inline Example/example.ts:99]: can remove these logs

Comment (daganomri) [reply Example/example.ts]: I removed them.

File: Example/example.ts [hunk 2]
```diff
@@ -32,15 +32,15 @@ async function example() {
     // loads the auth file credentials if present
```
Comment (adiwajshing) [inline Example/example.ts:32]: Can remove changes from example.ts -- it's only a prettier run on it

Comment (daganomri) [reply Example/example.ts]: I removed the prettier formatting.

**Pull request merged**